### ISSUE TITLE
[Bug] Window can be closed with Ctrl+W

### ISSUE THREAD

**Issue by SourLemonJuice** [Vencord/Vesktop#934]
### Operating System

Arch Linux

### Linux Only ~ Desktop Environment

KDE Plasma(Wayland)

### Package Type

AUR and Appimage, version 1.5.3

### What happens when the bug or crash occurs?

The window closes when I press Ctrl+W, but the official discord client does not.
It happened on any page, whether it's on the login or chat page.

### What is the expected behaviour?

Nothing will happen.
Blocking browser shortcut keys shouldn't be that hard, I guess.

### How do you recreate this bug or crash?

1. open a Vesktop window.
2. press Ctrl+W.

### Debug Logs

No new logs have been added, which is a very small thing.

### Request Agreement

- [X] I have searched the existing issues and found no similar issue
- [X] I am using the latest Vesktop and Vencord versions
- [x] This issue occurs on an official release (not just the AUR or Nix packages)

**Comment by yllekz** [comment]
Is this truly a bug? IMO this should be a standard feature for all apps, and quite often is. on macOS, it's a global system shortcut to CMD + W to close a window, and on Linux Mint, nearly every app I use defaults to this. I don't think it's a bug, and rather, I enjoy that this exists. The fact that the official Discord app doesn't do this sounds like a problem on their end that they should rectify.

**Comment by hir0xygen** [comment]
I agree with @yllekz, this definitely isn't a bug and I use this quite a lot.
It's pretty standard in lots of programs, a notable example possibly being the Nemo file explorer.
If this behavior were to be changed, maybe just via a toggle to block browser shortcuts?

**Comment by technikker** [comment]
Closing a window/tab/file via the Ctrl-W shortcut is an intentional feature in most apps.

In most cases it is an 'Alt-F4 Lite', which will close the currently opened file (e.g. notepad++) or tab (e.g. firefox), while leaving the app itself running, but some apps where the former case doesn't make sense will simply exit entirely.

**Comment by justDarian** [comment]
Respectfully, ive been using CTRL+W for a bit with Vesktop. Its the shortcut i use to close Vesktop! Please dont remove it, or atleast make its functionality toggable..

**Comment by yllekz** [comment]
File Explorer did respond to CTRL W long before Windows 11. I think as far back as Windows 7 even.

And also as a reminder, CMD W to close a window has been a thing in macOS for decades. This is not a new thing.

**Comment by Dwang-ML** [comment]
This should be kept as a feature, it is useful in many occasions and I appreciate the presence of this feature.
